src/atoms/buttons/Button.js

```diff
@@ -8,34 +8,44 @@ function Button ( props ) {
     children,
     onClick,
     shake,
-    className
+    className,
+    buttonType,
+    disabled
   } = props;
 
   return (
-    <div
+    <button
       className={ classnames(className, styles[type], { [styles.shaking]: shake }) }
       onClick={ onClick }
-    >{children}</div>
+      type={ buttonType }
+      disabled={ disabled }
+    >
+      {children}
+    </button>
   )
 }
 
 Button.propTypes = {
   /**
    * Possible button types are: `info, disabled, action, lowlight, actionDisabled, outline, or default (button)`
-   *
    */
   type: PropTypes.string.isRequired,
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
   /**
    * Will append new classname to classSet
    */
   className: React.PropTypes.string,
-  shake: PropTypes.bool
+  shake: PropTypes.bool,
+  /**
+   * `Submit, reset, button, or menu`
```

Comment:
@cisacke Perhaps we should explicitly state that these are the options available.